packages/gamut/src/Form/Input.tsx

```diff
@@ -42,6 +44,10 @@ export interface InputWrapperProps extends InputProps {
       HTMLInputElement
     >
   >;
+  /**
+   * An custom icon svg from gamut-icons.
```

Comment:
please don't hate me
```suggestion
   * A custom icon svg from gamut-icons.
```